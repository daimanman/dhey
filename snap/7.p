{"C":10,"Z":"2","N":10,"Remark":"TEST TASK","Url":"","Method":"post","TaskId":7,"Type":"","FileId":"2","Status":0,"Err":"","QH":{}}