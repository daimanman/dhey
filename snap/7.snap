{"AvgTotal":15.0913483,"Fastest":0.5254295,"Slowest":3.9094003,"Average":1.50913483,"Rps":2.557281507466303,"AvgConn":0.06749559999999999,"AvgDNS":0.00080246,"AvgReq":0,"AvgRes":1.3488048000000001,"AvgDelay":0.09283443,"ConnMax":0,"ConnMin":0.0010032,"DnsMax":0,"DnsMin":0.0010032,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.3279042,"DelayMax":0.0651543,"DelayMin":0.1453669,"Lats":[0.5254295,0.5394356,0.7860917,0.84027,1.1089543,1.1320144,1.5611576,1.7265979,2.961997,3.9094003],"ConnLats":[0.0010032,0.0010022,0.0010032,0.0010032,0.0010032,0,0.0010032,0,0.0010032,0.0010032],"DnsLats":[0.0010032,0.0010022,0.0010032,0.0010032,0.0010032,0,0.0010032,0,0.0010032,0.0010032],"ReqLats":[0,0,0,0,0,0,0,0,0,0],"ResLats":[0.3279042,0.4050777,0.5695152,0.702904,0.8993974,1.0066797,1.4247832,1.5952388,2.7835219,3.7730259],"DelayLats":[0.1293388,0.0671509,0.1453669,0.0661564,0.1383473,0.0691846,0.0681879,0.0651543,0.1102886,0.0691686],"Total":3910402500,"ErrorDist":{},"StatusCodeDist":{"200":10},"SizeTotal":1615537,"SizeReq":161553,"NumRes":10,"LatencyDistribution":[{"Percentage":10,"Latency":0.5394356},{"Percentage":25,"Latency":0.84027},{"Percentage":50,"Latency":1.1320144},{"Percentage":75,"Latency":2.961997},{"Percentage":90,"Latency":3.9094003},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.5254295,"Count":1,"Frequentcy":0.1},{"Mark":0.86382658,"Count":3,"Frequentcy":0.3},{"Mark":1.20222366,"Count":2,"Frequentcy":0.2},{"Mark":1.54062074,"Count":0,"Frequentcy":0},{"Mark":1.87901782,"Count":2,"Frequentcy":0.2},{"Mark":2.2174149,"Count":0,"Frequentcy":0},{"Mark":2.55581198,"Count":0,"Frequentcy":0},{"Mark":2.89420906,"Count":0,"Frequentcy":0},{"Mark":3.23260614,"Count":1,"Frequentcy":0.1},{"Mark":3.57100322,"Count":0,"Frequentcy":0},{"Mark":3.9094003,"Count":1,"Frequentcy":0.1}],"TaskId":7,"Duration":3.9104025}